src/rust/ensogl/lib/components/src/selector/number.rs

```diff
@@ -0,0 +1,134 @@
+use crate::prelude::*;
```

Comment:
no docs. Please check config why error was not emitted here.

Reply:
There are no errors here (and in the other places noted) because none of this is public. All of this are utility modules that are not exposed as part of the `selector` module. So, they are `public` only within the `selector` module, but not outside it, and thus they are counted as not public.